wire/core/ImageSizerEngine.php

```diff
@@ -889,6 +909,35 @@ public function setQuality($n) {
 		return $this;
 	}
 
+	/**
+	 * Set the image quality 1-100 for WebP output, where 100 is highest quality
+	 *
+	 * @param int $n
+	 *
+	 * @return $this
```

Comment:
Since we're in "code review mode" here, minor note about this: return statement should always state a valid return type, and `$this` is not one of those. `@return self` should work, although `@return ImageSizerEngine` would be recommended. Latter can be slightly confusing if and when this class is extended, so I'd probably go with `self` :)